3.4/x86/docker-compose.yml

```diff
@@ -16,7 +16,5 @@ moodle:
     - "80:80"
   links:
     - moodledb
```

Comment:
I think it is unnecessary since empty password is used if we don't define password

https://github.com/ole-vi/moodole/pull/2/files#diff-0330b6e080c20e45f520ade951d56ee2R11

```
- MYSQL_ALLOW_EMPTY_PASSWORD=yes
```